src/include/miopen/performance_config.hpp

```diff
@@ -40,7 +40,13 @@ namespace solver {
 
 struct PerfConfig
 {
-    virtual ~PerfConfig() = default;
+    PerfConfig() {}
+
+    virtual ~PerfConfig()         = default;
+    PerfConfig(const PerfConfig&) = default;
+    PerfConfig(PerfConfig&&)      = default;
+    PerfConfig& operator=(const PerfConfig&) = default;
+    PerfConfig& operator=(PerfConfig&&) = default;
```

Comment:
```suggestion
    virtual ~PerfConfig()         = default;
    ...
protected:
    PerfConfig() = default;
    PerfConfig(const PerfConfig&) = default;
    PerfConfig& operator=(const PerfConfig&) = default;
```

Reply:
Here you can add protected default constructor, protected copy constructor and protected copy assignment operator